play-json/shared/src/test/scala/WritesSharedSpec.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+import org.scalatest._
+
+class WritesSharedSpec extends WordSpec with MustMatchers {
```

Comment:
The `shared` part of the origin `WritesSpec` (without the tests about the JVM specific code)